packages/google_maps_flutter/google_maps_flutter_web/test/test_driver/shape_integration.dart

```diff
@@ -87,6 +90,29 @@ void main() {
       controller.update(options);
       verify(polygon.options = options);
     });
+
+    test('Polygon with hole has a hole', () {
+      final holedPolygon = platform.Polygon(
+        polygonId: platform.PolygonId('BermudaTriangle'),
+        points: [
+          platform.LatLng(25.774, -80.19),
+          platform.LatLng(18.466, -66.118),
+          platform.LatLng(32.321, -64.757),
+        ],
+        holes: [
+          [
+            platform.LatLng(28.745, -70.579),
+            platform.LatLng(29.57, -67.514),
+            platform.LatLng(27.339, -66.668),
+          ],
+        ],
+      );
+      final controller = PolygonsController(stream: null)
+        ..addPolygons({holedPolygon});
+      final gmapsPolygon = controller.polygons.values.first.polygon;
+      final pointInHole = gmaps.LatLng(28.632, -68.401);
+      expect(geometry.poly.containsLocation(pointInHole, gmapsPolygon), false);
+    });
```

Comment:
I think this test belongs in shape*s*_integration.dart, since it's using the Polygon*s*Controller. Can you move it there?

https://github.com/flutter/plugins/blob/master/packages/google_maps_flutter/google_maps_flutter_web/test/test_driver/shapes_integration.dart

(IIRC it was there before?)

Reply:
Moved it

Reply:
Thanks!!